lib/pocketape.dart

```diff
@@ -23,15 +25,20 @@ abstract class Pocketape {
           onDone: controller.close,
         );
         if (_count == 1) {
+          print("Start");
+          if (Platform.isAndroid && !(await _requestPermission())) {
```

Comment:
Minor: directly call `await Permission.contacts.request().isGranted` this also checks whether it has already been granted.

Reply:
@Robbendebiene _requestPermission() method is not only checking, but it is also checking and if they are not granted ask the user to approve, in that case, that line can not be replaced.

Reply:
`await Permission.camera.request().isGranted` should do exactly what we need (check and request when needed). Docs say:

> Call request() on a Permission to request it. If it has already been granted before, nothing happens.
request() returns the new status of the Permission.

```
if (await Permission.contacts.request().isGranted) {
  // Either the permission was already granted before or the user just granted it.
}
```